fl4health/utils/metrics.py

```diff
@@ -105,16 +105,32 @@ def __call__(self, logits: torch.Tensor, target: torch.Tensor) -> Scalar:
 
 
 class F1(Metric):
-    def __init__(self, name: str = "F1 score"):
+    def __init__(
+        self,
+        name: str = "F1 score",
+        average: Optional[str] = "weighted",
```

Comment:
Just adding an additional functionality to use different averaging approaches for the F1 calculations.